substrate/frame/identity/src/legacy.rs

```diff
@@ -24,6 +24,9 @@ use sp_std::prelude::*;
 
 use crate::types::{Data, IdentityFields, IdentityInformationProvider, U64BitFlag};
 
+#[cfg(feature = "runtime-benchmarks")]
+const MAX_ADDITIONAL_FIELDS: usize = 2;
```

Comment:
Why can we not use the correct maximum from the runtime?

Reply:
We surely can. I was getting rid of the `MaxAdditionalFields` instances from the configs and I forgot to get rid of this, good catch!